### PR TITLE
WIP: Enhancement grammatical edits

### PR DESCRIPTION
**Pull request by rbuccigrossi**
This pull includes grammatical edits on the lesson (starting with 01-shell-basics). 

I'm pushing this as a Work In Progress so that you can review it and make sure I'm doing the right thing :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -57,6 +57,23 @@ In brief:
...
+sudo gem install jekyll
+```
+
+Then from your git checkout you will be able to execute:
+
+```Bash
+make serve
+```
+
+which will generate and serve the lessons on <http://localhost:4000>.
```
Comment (yarikoptic) [inline CONTRIBUTING.md:75]: indeed!  although might be worth  to be introduced as well to
http://github.com/ReproNim/module-template
on which one this one is based

File: _episodes/01-shell-basics.md [hunk 2]
```diff
@@ -8,42 +8,43 @@ questions:
...
-  neuroimaging tools might use a different shell and thus provide
-  instructions which would not be compatible with your current shell"
-- "Command line shell is a powerful tool, and learning additional
+- "There are a number of incompatible shells. Different
+  neuroimaging tools may use specific shells and thus provide
+  instructions that are not compatible with your current shell."
+- "A command line shell is a powerful tool and learning additional
  'tricks' could help to make its use much more efficient, less
- error-prone, and thus more reproducible"
+ error-prone, and thus more reproducible."
```
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: as you might have noted, me is "greedy" (and somewhat consistent) on trailing periods if some other separator is used and it is clear that statement is 'done'.  That is why in this case, as well as among Objectives, I did not use trailing period.  In general -- I do not care much but OCD-me likes to be consistent

File: _episodes/01-shell-basics.md [hunk 3]
```diff
@@ -8,42 +8,43 @@ questions:
...
 - "Shell scripting is the most accessible tool to automate execution of
-  arbitrary set of commands to largely avoid manual retyping of the
-  same commands possibly leading to typos and erroneous analyses"
-- "Environment variables play a big role in defining script behavior"
+  arbitrary set of commands. This will avoid manual retyping of the
```
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: why `will` here? all the other statements are "in present" tense statements of the facts ;)

File: _episodes/01-shell-basics.md [hunk 4]
```diff
@@ -69,7 +70,7 @@ Debian) providing those helpful command line utilities.
 
 In this module we will first become familiarized with basic (and
 possibly advanced) features of a shell and shell scripting, and then
-review a few aspects which particularly relate to reproducibility, as
+review a few aspects t particularly relate to reproducibility, as
```
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: it is always valuable to review `git diff` or better `git diff --color-words` in this case before committing ;)

**Review by yarikoptic — CHANGES_REQUESTED** [review]
getting pulled to switch, will finish reviewing later - but here are some comments ;)

File: _episodes/01-shell-basics.md (hunk 2 above)
Comment (rbuccigrossi) [reply _episodes/01-shell-basics.md]: Hmmm... git makes it a bit harder for you to accept changes you like.  But I can't leave sentences without periods :)

File: _episodes/01-shell-basics.md (hunk 3 above)
Comment (rbuccigrossi) [reply _episodes/01-shell-basics.md]: Bah. I split the sentence, and went through a couple of iterations of changes, leaving the tense incorrect.

File: _episodes/01-shell-basics.md (hunk 2 above)
Comment (rbuccigrossi) [reply _episodes/01-shell-basics.md]: The other issue is that in order to make the key points easier to read, I split a couple into multiple sentences. But then not having a period at the end of the last sentence looks odd, and then I felt compelled to add periods at the end of the rest of the key points in order to have parallel structure.

File: _episodes/01-shell-basics.md (hunk 4 above)
Comment (rbuccigrossi) [reply _episodes/01-shell-basics.md]: Hmmm... I'm trying to go further and use "Remarkable" with a spell checker, but it doesn't seem to be completely reliable.

File: _episodes/01-shell-basics.md (hunk 2 above)
Comment (rbuccigrossi) [reply _episodes/01-shell-basics.md]: OK. I removed the periods and separated the sentences with a semicolon.  BTW, git diff --color-words is wonderful.

Comment (yarikoptic) [reply _episodes/01-shell-basics.md]: yeah -- saw semicolons ;-)  I will let them be although I would have just kept periods interim sentences.
you might then like also following aliases. 
```
    wdiff = diff --color-words
    pdiff = diff 'HEAD@{1}'
    pdfdiff = difftool -x diffpdf --no-prompt
    niidiff = difftool -x niidiff --no-prompt
    kompare = difftool -x kompare --no-prompt
    nbdiff = difftool -x nbdiff --no-prompt
```
I do use kompare also from time to time if I want quick GUI view and there is not that many files changed

File: index.md [hunk 5]
```diff
@@ -3,14 +3,14 @@ layout: lesson
 ---
 
 
-*Reproducibility* could not only be enforced via dedicated systems
-facilitating automated and reproducible computation, but also could
-be taken as a principle to adhere to while performing any day-to-day
+The term "reproducibility" conjures a mental image of dedicated systems
+conducting automated and repeatable computations.  However, **you** can
+embrace reproducibility as a principle to apply to your day-to-day
```
Comment (yarikoptic) [inline index.md:8]: lovely!

**Review by yarikoptic — APPROVED** [review]

**Pull request merged**